plugins.json

```diff
@@ -889,5 +889,17 @@
 		"min_version": "4.8.0",
 		"variant": "both",
 		"creation_date": "2024-04-1"
+	},
+	"cosmic_reach_model_editor": {
+		"title": "Cosmic Reach Model Editor",
+		"author": "Z. Hoeshin",
+		"icon": "icon.png",
+		"description": "Allows creating, editing, importing and exporting Cosmic Reach block models.",
+		"tags": ["Cosmic Reach"],
+		"version": "1.1.0",
+		"min_version": "4.9.4",
+		"variant": "both",
+		"creation_date": "2024-03-09",
+		"new_repository_format": true,
```

Comment:
The plugin's meta data is mismatched with the meta data defined in the code. (mismatched `tags`, missing `min_version`, `variant`, `creation_date`, `new_repository_format`)
```js
    Plugin.register(id, {
      title: name,
      icon: "icon.png",
      author: "Z. Hoeshin",
      description: "Allows creating, editing, importing and exporting Cosmic Reach block models.",
      tags: [],
      version: "1.1.0",
      creation_date: "2024-03-09",
```